test/it/negative.spec.js

```diff
@@ -1,73 +0,0 @@
-const compiler = require("../utils/compiler");
```

Comment:
i guess you deleted the test files by accident ?

Reply:
IDK man it's still there... maybe it was a mistake made in the commit split somehow.
Final result is that it is there